Modules/SubModule_Import/xASL_imp_DCM2NII_Subject_ShuffleTheDynamics.m

```diff
@@ -1,9 +1,10 @@
-function [nii_files, summary_line, globalCounts, ASLContext] = xASL_imp_DCM2NII_Subject_ShuffleTheDynamics(globalCounts, scanpath, scan_name, nii_files, iSubject, iSession, iScan)
+function [x,nii_files, summary_line, globalCounts, ASLContext] = xASL_imp_DCM2NII_Subject_ShuffleTheDynamics(x,globalCounts, scanpath, scan_name, nii_files, iSubject, iSession, iScan)
```

Comment:
This function resorts the ASL volumes right? "dynamics" is a specific Philips term, would replace this by "volumes". Also, would replace the function name "shuffle" by "sort" (shuffling has a connotation as if it happens at random, e.g., "shuffling cards")

Reply:
Yes, @jan-petr and me talked about renaming that script for a while. Let's do this afterwards in a separate issue, ok?